src/main/java/com/team4/testingsystem/entities/ErrorReport.java

```diff
@@ -30,6 +30,10 @@
     @JoinColumn(name = "test_id")
     private Test test;
```

Comment:
What do you think about the possibility to use a pair of testId and questionId as a PK instead of building meaningless IDs?

Reply:
Maybe we can use [class like this](https://github.com/AntonMaksimenkau/team4_backend/pull/49/files#diff-7af3f91cccd82763ac02a1ca9c9c420fc1b93153e0cf5075a7c72a2211fb7298R9) as an ID not only for chosen option? We use a pair (question_id, test_id) almost everywhere